tests/NuGet.Services.AzureSearch.FunctionalTests/Analysis/DescriptionCustomAnalyzerFunctionTests.cs

```diff
@@ -81,6 +81,10 @@ public static IEnumerable<object[]> ProducesExpectedTokensData()
                         "hello"
                     }
                 },
+
+                // Unicode surrogate pairs
```

Comment:
Consider combining `DescriptionCustomAnalyzerFacts.ProducesExpectedTokensData` and `PackageIdCustomAnalyzerFacts.ProducesExpectedTokensData` to reduce repetition.

Reply:
Good suggestion, done!